include/lsqecc/ls_instructions/ls_instructions.hpp

```diff
@@ -75,15 +75,15 @@ struct MagicStateRequest {
     PatchId target;
     PatchId near_patch; // TODO make optional for backwards compatibility
     
-    static const size_t DEFAULT_WAIT = 100;
+    static const size_t DEFAULT_WAIT = std::numeric_limits<size_t>::max();
```

Comment:
Watch out the purpose of this is to error out after a bit. Is the intention to always set it manually?

Reply:
Maybe there should be some sort of setting to control this. Especially, when debugging it might be helpful to lower DEFAULT_WAIT.

Reply:
The issue is that, in the dag or wave pipelines, when there are many more heads than what can actually be routed, sometimes it can take a long time but it shouldn't fail